world-chain-builder/crates/world/node/benches/validate_transaction.rs

```diff
@@ -1,192 +0,0 @@
-use std::future::Future;
```

Comment:
It might be nice to keep this, so we can get an estimate on how validation time scales with large bundles

Reply:
yeah, I only removed it because it seemed to deal with the previous PBH setup. I def want to have benches but let's re-add them once we're done with the builder code imo